### DISCUSSION TITLE
Makefile.shared missing in openssl-1.1.1u

### DISCUSSION THREAD

**Discussion by vsrandom** [openssl/openssl#22513] [Q&A] [answered]
Hi. In my organization we use Openssl1.1.0f by obfuscation  (replacing functions and variable names with random names). 
Now we have to upgrade to Openssl-1.1.1u. 

Version "1.1.0f" had Makefile.shared which used crypto.map and ssl.map to build libcrypto.so and libssl.so
We used to create new crypto_a.map and new ssl_a.map with changed names and then used it to build the openssl.

Now Openssl 1.1.1u doesn't contain Makefile.shared, so our previous implementation is not working. 

Kindly guide me how can this be achieved. I'm new to building stuff in linux so any help will be highly appreciated.

@levitte

**Comment by paulidale** [comment]
Version 1.1.1 is out of support.

Nonetheless, I suggest reading and following the install instructions in the INSTALL.md file.

**Comment by mspncp** [comment, marked as answer]
TL;DR: you need to run the `./Configure` command (with appropriate arguments), which will create a `Makefile` (without the `.shared` extension) for unix-ish targets.
